proto/toit/api/auth.proto

```diff
@@ -41,3 +42,12 @@ message ChangeOrganizationRequest {
   bytes organization_id = 1;
   bytes refresh_token = 2; // optional
 }
+
+message CreateOrganizationRequest {
+  string org_name = 1;
```

Comment:
just name

Reply:
Alternative would be to seperate it into to sections:
```
message CreateOrganizationRequest {
  message Organization {
    string name = 1;
  }
  message User {
    string email = 1;
  }
  Organization organization = 1;
  User user = 1;
}
``` 
That way it would be easy to distinquish for the API developer and if we introduce a user name at some point it becomes easy to create.